contributing/code/standards.rst

```diff
@@ -101,33 +109,34 @@ short example containing most features described below::
             );
 
             if (true === $dummy) {
-                return null;
+                return 'something';
             }
 
-            if ('string' === $dummy) {
+            if (is_string($dummy)) {
                 if ('values' === $mergedOptions['some_default']) {
                     return substr($dummy, 0, 5);
                 }
 
                 return ucwords($dummy);
             }
+
+            return null;
         }
 
         /**
-         * Performs some basic check for a given value.
+         * Performs some basic operations for a given value.
          *
-         * @param mixed $value     Some value to check against
+         * @param mixed $value     Some value to operate against
          * @param bool  $theSwitch Some switch to control the method's flow
-         *
-         * @return bool|void The resultant check if $theSwitch isn't false, void otherwise
```

Comment:
Note: I have seen *some* `@return void` in the code, but below is explicitly stated
> Omit the ``@return`` tag if the method does not return anything